test/unit/gateways/stripe_test.rb

```diff
@@ -342,8 +342,7 @@ def test_successful_refund
     assert response = @gateway.refund(@refund_amount, 'ch_test_charge')
     assert_success response
 
-    assert_equal 'ch_test_charge', response.authorization
-    assert response.test?
```

Comment:
Stripe's Refund object doesn't have a `livemode` flag to use when populating `test?`. Suggestions?
https://stripe.com/docs/api#refund_object